filter/Filters.hpp

```diff
@@ -392,6 +412,9 @@ static void decompressFile(const Shared *const shared, const char *filename, FMo
   } else {
     printf("done   \n");
   }
+#else
+// TODO: does `f.getchar()` have any side effects? Does it still _need_ to be called?
```

Comment:
@GotthardtZ I think it's safe to remove the call to `f.getChar()` but I wasn't sure.